classes/WP_CLI/Action/Create.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+use function \WP_CLI\Utils\get_flag_value;
+
+/**
+ * WP-CLI command: action-scheduler action create
+ */
+class ActionScheduler_WPCLI_Action_Create_Command extends ActionScheduler_WPCLI_Command {
```

Comment:
I am getting some issues using this command, seems like we are somehow setting the `function_args` to an int

```
> wp action-scheduler action create woocommerce_cleanup_draft_orders now

Fatal error: Uncaught TypeError: Argument 2 passed to ActionScheduler_Action::__construct() must be of the type array, int given, called in /usr/src/public_html/wp-content/plugins/action-scheduler/classes/ActionScheduler_ActionFactory.php on line 314 and defined in /usr/src/public_html/wp-content/plugins/action-scheduler/classes/actions/ActionScheduler_Action.php:26
Stack trace:
#0 /usr/src/public_html/wp-content/plugins/action-scheduler/classes/ActionScheduler_ActionFactory.php(314): ActionScheduler_Action->__construct('woocommerce_cle...', 10, Object(ActionScheduler_NullSchedule), '')
#1 /usr/src/public_html/wp-content/plugins/action-scheduler/functions.php(51): ActionScheduler_ActionFactory->create(Array)
#2 /usr/src/public_html/wp-content/plugins/action-scheduler/classes/WP_CLI/Action/Create.php(82): as_enqueue_async_action('woocommerce_cle...', 10)
```